### PR TITLE
Add xlsx support

### PR DESCRIPTION
**Pull request by abhistar**

### CODE REVIEW

File: src/main/java/com/echainika/app/controller/AdminController.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        }
+        catch (IllegalArgumentException e) {
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+        }
+        catch (Exception ex) {
+            return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+    }
+
+    @PatchMapping("/editCandidate")
```
Comment (anushreejain98) [inline src/main/java/com/echainika/app/controller/AdminController.java]: fix the api path, remove create, update etc from url

File: src/main/java/com/echainika/app/model/Error.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.echainika.app.model;
+
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class Error {
+    private Integer rowNumber;
```
Comment (anushreejain98) [inline src/main/java/com/echainika/app/model/Error.java:10]: should be candidate id

File: src/main/java/com/echainika/app/model/CandidateAndErrors.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+import com.echainika.app.model.dto.request.CandidateRequest;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Builder
+public class CandidateAndErrors {
```
Comment (anushreejain98) [inline src/main/java/com/echainika/app/model/CandidateAndErrors.java]: rename .

File: src/main/java/com/echainika/app/controller/AdminController.java (hunk 1 above)
Comment (abhistar) [reply src/main/java/com/echainika/app/controller/AdminController.java]: resolved

File: src/main/java/com/echainika/app/model/Error.java (hunk 2 above)
Comment (abhistar) [reply src/main/java/com/echainika/app/model/Error.java:10]: cannot be renamed as it signifies the row number in the excel sheet which was uploaded to update candidate information in the db

**Pull request merged**